spec/support/itemizable_shared_example.rb

```diff
@@ -5,6 +5,26 @@
   context ".items" do
   end
 
+  context ".to_a" do
```

Comment:
Nitpick: If the context is just the name of the method, it's more idiomatic to use `describe`.